### PR TITLE
Small fixes in plots

### PR DESCRIPTION
**Pull request by jakub-piw**
Bar plots are now horizontal, the ensemble model is highlighted on metrics plots, compatimetrics matrices were given clear hover texts.

### CODE REVIEW

**Review by D3TR0 — APPROVED** [review]
Code works and looks good

File: components/compatimetrics_plots.py [hunk 1]
```diff
@@ -85,11 +92,17 @@ def rmsd_matrix(predictions):
                 matrix[i, j] = round(root_mean_squared_difference(predictions[models[i]],
                                                                   predictions[models[j]]))
     matrix = pd.DataFrame(matrix, index=models, columns=models)
-    fig = px.imshow(matrix, text_auto=True, color_continuous_scale=[[0, 'lightblue'], [0.5, 'blue'], [1, 'purple']])
+    fig = px.imshow(matrix, color_continuous_scale=[[0, 'lightblue'], [0.5, 'blue'], [1, 'purple']],
+                    labels=dict(x="Model 1",
```
Comment (malwina0) [inline components/compatimetrics_plots.py:96]: In app I can't see this x-axis legend. It is only on the y-axis it is model 2. Fix it or remove the legend from the y-axis.

**Review by malwina0 — CHANGES_REQUESTED** [review]
Overall great, only this one small bug.

File: components/compatimetrics_plots.py (hunk 1 above)
Comment (jakub-piw) [reply components/compatimetrics_plots.py:96]: Fixed

**Review by malwina0 — APPROVED** [review]
I've corrected it myself

**Pull request merged**